### PR TITLE
Remove unused jupyter functionality

### LINKED ISSUE #140: Docker instructions don't work
The codebase has changed quite a lot since the beginning in 2022. We no longer use the ipython notebooks, and we also have separate scripts for daily and monthly plots. This has caused the docker script to no longer work. To fix this we should first of all remove all of the unnecessary ipython notebooks as they're no longer used, and then update the docker script to work with the new codebase. I've started to work on this in #118.

### PR DESCRIPTION
**Pull request by huaracheguarache**
Could you help me take a look if there's anything else we can remove, @epifanio?

Resolves #140.

### CODE REVIEW

**Comment by huaracheguarache** [comment]
I just ran a test trying to build and bring up the docker build, and it doesn't work. We need to clean up the repo by removing all jupyter-related functionality because it's not used anymore, and we also need to change the docker files to work with the new build. Could you help me with that, @epifanio?

**Comment by huaracheguarache** [comment]
Just a note for the future: we'll also need to update the README instructions.

File: Dockerfile
Comment (epifanio) [inline Dockerfile:1]: you can try to override the `COPY` statement and replace the `CMD` with a call to an `entrypoint` script


sample `entrypoint.sh` 

```bash
#!/bin/sh
panel serve --port ${PORT} --address 0.0.0.0 --allow-websocket-origin ${ORIGIN} --log-level ${LOG_LEVEL} /app

```

then in the `dockerfile`:

```dockerfile
COPY entrypoint.sh /entrypoint.sh
RUN chmod +x /entrypoint.sh

EXPOSE 7000

ENV ORIGIN="0.0.0.0:7000" PORT="7000" PREFIX="" LOG_LEVEL="debug"

COPY ./app /app
ENTRYPOINT ["./entrypoint.sh"]
```
Change the `COPY` reflecting the repository structure

then the `docker_compose.yml` can be like:

```yaml
services:
  iceplot:
    image: epinux/iceplot:latest
    build:
      context: ./docker/iceplot
    environment:
      BOKEH_ALLOW_WS_ORIGIN: 'localhost:7778,0.0.0.0:7778,192.168.176.1:7778'
      PORT: '7777'
      LOG_LEVEL: 'debug'
      BOKEH_LOG_LEVEL: 'debug'
      PYTHONUNBUFFERED: 1
    ports:
      - 7778:7777
    hostname: iceplot
    volumes:
      - ./app:/app
      - ./docker/iceplot/entrypoint.sh:/entrypoint.sh
 ```
 
 Change the `PATH` according with the repository structure

File: compose.yaml
Comment (epifanio) [inline compose.yaml:1]: It is good to have the code exposed as a volume and the command to start the service as an entrypoint, see the comments in the docker file section, sample `docker-compose.yml` can look like:

```yaml
services:
  iceplot:
    image: epinux/iceplot:latest
    build:
      context: ./docker/iceplot
    environment:
      BOKEH_ALLOW_WS_ORIGIN: 'localhost:7778,0.0.0.0:7778,192.168.176.1:7778'
      PORT: '7777'
      LOG_LEVEL: 'debug'
      BOKEH_LOG_LEVEL: 'debug'
      PYTHONUNBUFFERED: 1
    ports:
      - 7778:7777
    hostname: iceplot
    volumes:
      - ./app:/app
      - ./docker/iceplot/entrypoint.sh:/entrypoint.sh
```

File: requirements.txt
Comment (epifanio) [inline requirements.txt:1]: make sure to use the latest available version (even for python) as much as possible :)

**Review by epifanio — COMMENTED** [review]
I made some comments on the Dockerfile and docker-compose

**Comment by huaracheguarache** [comment]
Bump @epifanio. Could you look over the docker files to see whether there's something I should change. I want you to be able to use them out of the box for the server if possible, since that would make life a bit easier.

**Review by epifanio — APPROVED** [review]
Great cleaning! I think is good to go

**Pull request merged**